__tests__/praxis/arthur-deviation-biases/unit.ts

```diff
@@ -34,14 +33,16 @@ describe('ArthurDeviationBiases Class: Unit', () => {
           momentum: 0.2,
         },
       });
-      const result = kernel(weights, deltas);
-      expect(shave(result)).toEqual(shave([[1.25, 2.20000005, 3.1500001]]));
+      const result: any = kernel(weights, deltas);
```

Comment:
Use type for `result`

Reply:
same problem here `Argument of type 'KernelOutput' is not assignable to parameter of type 'Float32Array'.
  Type 'number' is not assignable to type 'Float32Array'.` on line 38.

Reply:
`same problem here Argument of type 'KernelOutput' is not assignable to parameter of type 'Float32Array'. Type 'number' is not assignable to type 'Float32Array'`. on line 38.

Reply:
You can cast it to `Float32Array` for now. 

This requires using generics in the base neural network class, which will be covered in another pull request.

Reply:
done...